plugins/catalog-common/api-report.md

```diff
@@ -0,0 +1,37 @@
+## API Report File for "@backstage/plugin-catalog-common"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { Permission } from '@backstage/plugin-permission-common';
+
+// @public
+export const catalogEntityReadPermission: Permission;
+
+// @public
+export const catalogEntityRefreshPermission: Permission;
+
+// @public
+export const catalogEntityUnregisterPermission: Permission;
```

Comment:
This is actually "delete" rather than "unregister" right? At least that's how I picture it in my mind.

Reply:
Done.